librad/src/git/trailer.rs

```diff
@@ -0,0 +1,139 @@
+// This file is part of radicle-link
+// <https://github.com/radicle-dev/radicle-link>
+//
+// Copyright (C) 2019-2020 The Radicle Team <dev@radicle.xyz>
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 3 or
+// later as published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+use std::convert::TryFrom;
+
+#[derive(Debug, Clone, Eq, PartialEq, thiserror::Error)]
+pub enum Error {
+    #[error("the given input is not a well-formed trailer")]
+    IllFormed,
```

Comment:
nit: We have been using sentence case since before radicle-dev/radicle-decisions#3 landed. I don't have a strong preference for either style, but I think this might be one of the very rare cases where consistency is preferable. Perhaps we can have a separate patch which converts everything to lower case in one go, if that's the preferred style?

Reply:
I am good with your argument for consistency. The standard to follow is to go lower case, the reason for that being that when building a stack of errors messages, having them lower case shows that continuity. We have adopted it fully in the registry and it's a good looking approach indeed.

I will make this now consistent with the other error messages in this repo and create a follow-up issue to make them all lower case.